### PR TITLE
Added method to reassign ownership for Postgres multi-user-rotation.

### PR DESCRIPTION
**Pull request by FloWi**
I created an alternative solution to the ownership problem for the multi-user-rotation.
The solution provided by @thekevinbrown in issue #39 didn't work in our case.

https://github.com/aws-samples/aws-secrets-manager-rotation-lambdas/pull/39#issuecomment-646551901

*Issue #, if available:*
Issue #38 
*Description of changes:*
I found an easy way to transfer ownership of all objects created by one user to another and created this PR. 

**I didn't have a chance to test this yet**, since I didn't have time to set up a local environment and I'm not a python developer. So, please keep that in mind and take a closer look at my code :) 
Do you have a test-environment for your rotation-lambdas?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by xsnrg** [comment]
@FloWi Did you find another path to solve this, or are you just using your branch code for the lambda? 

This is a needed fix. 

@followben and @madsid

**Comment by FloWi** [comment]
@xsnrg I think we discarded this feature for now because of this bug. We're not live yet so no need for password rotation.

**Review by selfvin — CHANGES_REQUESTED** [review]

File: SecretsManagerRDSPostgreSQLRotationMultiUser/lambda_function.py [hunk 1]
```diff
@@ -198,6 +198,22 @@ def set_secret(service_client, arn, token):
...
+                reassign_ownership(current_dict, pending_dict['username'])
+    finally:
+        conn.close()
+
+
+def reassign_ownership(current_owner_dict, new_owner_username):
+    conn = get_connection(current_owner_dict)
+    try:
+        with conn.cursor() as cur:
+            cur.execute("REASSIGN OWNED BY %s TO %s" % (current_owner_dict['username'], new_owner_username))
```
[the remark this answers is not in the document]
Comment (FloWi) [reply SecretsManagerRDSPostgreSQLRotationMultiUser/lambda_function.py]: Sure thing. Thanks for the hint!

[the remark this answers is not in the document]
Comment (FloWi) [reply SecretsManagerRDSPostgreSQLRotationMultiUser/lambda_function.py]: Let's see if anyone at aws cares about that stuff :)

**Pull request closed without merging**